database/migrations/2017_11_20_202525_add_quantity_column_to_posts_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddQuantityColumnToPostsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('posts', function (Blueprint $table) {
+            $table->integer('quantity')->after('northstar_id')->default(null)->nullable();
```

Comment:
I don't think you need to set the `default(null)` explicitly because the default of `nullable()` columns is null

Reply:
@sbsmith86 ah yeah you're right!